src/plugins/intel_cpu/src/nodes/batch_to_space.cpp

```diff
@@ -206,36 +206,36 @@ void BatchToSpace::batchToSpaceKernel() {
             finish[3] = (outShape5D[3] + blockShape[3] - 1 - oAdd[3]) / blockShape[3];
             begin[4] = (blockShape[4] - 1 - oAdd[4]) / blockShape[4];
             finish[4] = (outShape5D[4] + blockShape[4] - 1 - oAdd[4]) / blockShape[4];
-            const int64_t addTmpOC = blocked ? 0LU : oAdd[1];
-            const int64_t addTmpOc = blocked ? oAdd[1] : 0LU;
+            const int64_t addTmpOC = blocked ? 0 : static_cast<int64_t>(oAdd[1]);
+            const int64_t addTmpOc = blocked ? static_cast<int64_t>(oAdd[1]) : 0;
 
             const size_t firstI1 = i0 == 0 ? std::max(begin[1], indxStart[1]) : begin[1];
             const size_t lastI1 = i0 == indxEnd[0] ? std::min(indxEnd[1] + 1, finish[1]) : finish[1];
 
             for (size_t i1 = firstI1; i1 < lastI1; ++i1) {
                 const size_t block = i1 == finish[1] ? lastBlock : blockSize;
-                const int64_t tmpOC = i1 * blockShape[1] + addTmpOC;
+                const int64_t tmpOC = static_cast<int64_t>(i1 * blockShape[1]) + addTmpOC;
```

Comment:
this is a conversion from 'size_t', isn't it? Why do we still need to have explicit cast after we basically disabled the check for this type?

Reply:
Re-applied with updated ignore type (added `unsigned long`, `unsigned int` there)